dll/ntdll/ldr/ldrapi.c

```diff
@@ -1658,4 +1658,19 @@ LdrFlushAlternateResourceModules(VOID)
     return FALSE;
 }
 
+/*
+ * @unimplemented
+ * See https://www.kernelmode.info/forum/viewtopic.php?t=991
+ */
+NTSTATUS
+NTAPI
+LdrSetAppCompatDllRedirectionCallback(
```

Comment:
I never looked at this because in vista+ it's gone.
Do you have plans with this?

Reply:
No, I just googled some functions for their prototypes and since I found this, I added a stub.